lib/rubocop/cop/style/unneeded_percent_q.rb

```diff
@@ -14,6 +14,7 @@ def on_dstr(node)
         end
 
         def on_str(node)
+          return unless standalone_string?(node)
```

Comment:
I will make the change to the method name and add comments. I will also look into refactoring this cop to implement different checks.